OpenRA.Game/Server/Connection.cs

```diff
@@ -90,6 +90,25 @@ public void ReadData(Server server)
 					var bytes = PopBytes(ExpectLength);
 					switch (State)
 					{
+						case ReceiveState.Handshake:
+							{
+								byte min = bytes[0];
+								byte max = bytes[1];
+
+								// We only support a single version at the moment but other implementations might support more.
+								if (min > ProtocolVersion.Version || ProtocolVersion.Version > max)
+								{
+									Log.Write("server", "Dropping client {0} for unsupported version: Server={1}, Client Min={2}, Client Max={3}",
+											PlayerIndex, ProtocolVersion.Version, min, max);
+									server.DropClient(this);
+								}
+								else
+								{
+									ExpectLength = 8;
+									State = ReceiveState.Header;
+								}
+							} break;
```

Comment:
Please put this `break` inside the braces, and would you mind fixing the other two cases in this switch too?

This then lets me drop a couple more style fixes from my pending StyleCop PR.

Reply:
I dropped the braces completely, it looks better that way and also matches how switch is used in other places.